modules/claims_api/app/clients/claims_api/bgs_client/definitions.rb

```diff
@@ -109,6 +109,77 @@ module FindBenefitClaimsStatusByParticipantId
         end
       end
 
+      ##
+      # IntentToFileWebServiceBean
+      #
+      module IntentToFileWebServiceBean
+        DEFINITION =
+          Bean.new(
+            path: 'IntentToFileWebServiceBean',
+            namespaces: Namespaces.new(
+              target: 'http://intenttofile.services.vetsnet.vba.va.gov/',
+              data: nil
+            )
+          )
+      end
+
+      module IntentToFileWebService
+        DEFINITION =
+          Service.new(
+            bean: IntentToFileWebServiceBean::DEFINITION,
+            path: 'IntentToFileWebService'
+          )
+
+        module InsertIntentToFile
+          DEFINITION =
+            Action.new(
+              service: IntentToFileWebService::DEFINITION,
+              name: 'insertIntentToFile',
+              key: 'IntentToFileDTO'
+            )
+        end
+
+        module FindIntentToFileByPtcpntIdItfTypeCd
+          DEFINITION =
+            Action.new(
+              service: IntentToFileWebService::DEFINITION,
+              name: 'findIntentToFileByPtcpntIdItfTypeCd',
+              key: 'return'
+            )
+        end
+      end
```

Comment:
technically I think we just need the bean and service on each of these for these to work, but we can provide actions for fun - may just come back and wipe these out eventually.

Reply:
Yea, I saw he had actions in the ones he started with so I figure I'd continue the pattern. If I'm seeing his vision correctly, it could make things simpler down the road 🤷